checkstyle.xml

```diff
@@ -51,6 +51,11 @@
 		<property name="max" value="1500"/>
 		<property name="severity" value="error"/>
 	</module>
+    
+    <module name="LineLength">
```

Comment:
i actually notice that these are the same changes that you deleted

Reply:
Refer to my comment [above](https://github.com/openmrs/openmrs-core/pull/4077#discussion_r865564296)